module/extension/type/processing/src/test/java/org/openbase/jul/extension/type/processing/LabelProcessorTest.kt

```diff
@@ -49,6 +49,26 @@ class LabelProcessorTest {
         Assertions.assertEquals("Living Cei Ling Lamp", LabelProcessor.format("Living   CeiLing      Lamp"))
     }
 
+    @Test
+    fun formatLabel() {
+        println("formatLabel")
```

Comment:
Maybe rename to something more specific such as `test format removes language with empty labels`